Resources/Prototypes/DeltaV/Body/Organs/vulpkanin.yml

```
@@ -4,6 +4,20 @@
   noSpawn: true
   components:
   - type: Stomach
+    poorlyDigestibleFood:
+      reducedReagents:
+        - Nutriment
+        - Vitamin
+      whitelistTags:
+        - Fruit # LostParadise's lore states the vulpkanin race isn't really adapted to plant food
```

yamllint

[warning] too few spaces before comment

(comments)